src/registries/functionsRoutersRegistry.ts

```diff
@@ -0,0 +1,68 @@
+export const functionsRoutersRegistry = {
+  ethereum: {
+    contractAddress: "0x65Dcc24F8ff9e51F10DCc7Ed1e4e2A61e6E14bd6",
+    donId: "fun-ethereum-mainnet-1",
+    donIdHex:
+      "0x66756e2d657468657265756d2d6d61696e6e65742d3100000000000000000000",
+    encryptedSecretsUploadEndpoints: [
```

Comment:
recommend naming consistency with CL Funcs documentation:
`encryptedSecretsUploadGateways`

Reply:
Yes, you are right. It will be more clear. That name "encryptedSecretsUploadEndpoints" came from here https://docs.chain.link/chainlink-functions/supported-networks.